packages/itwinui-react/src/styles.js/classes.mjs

```diff
@@ -2,6 +2,22 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-import styles from './styles.module.css';
 
-export default styles;
+// Side-effect import to ensure CSS file is generated even though the classes are not used.
+import './styles.module.css';
+
+// This proxy converts `iui-` to `_iui3-` in class names with very little code.
+// This is more efficient than exporting the entire mapping of original-to-transformed classes.
+export default new Proxy(
+  {},
```

Comment:
I'm a bit confused. How is this object populated if it starts off as empty?

Reply:
the object is never populated. when you try to run `styles['whatever']` it simply runs the `get` function

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy